src/rules/no-loud-comments/README.md

```diff
@@ -0,0 +1,27 @@
+# no-loud-comments
```

Comment:
The name should be `comment-no-loud` to be consistent with stylelint:
https://github.com/stylelint/stylelint/blob/master/docs/user-guide/rules.md#comment